index.html

```diff
@@ -24,5 +53,6 @@
                 <li><a href="https://github.com/elementary" target="_blank"><i class="fa fa-github"></i></a></li>
             </ul>
         </footer>
+        <script src="js/carousel.min.js"></script>
```

Comment:
If you're only now loading the script, what does the content look like before? How does it look without JavaScript?